src/Hosting/Hosting/src/Host.cs

```diff
@@ -8,11 +8,45 @@
 
 namespace Microsoft.Extensions.Hosting
 {
+    /// <summary>
+    /// Provides convenience methods for creating instances of <see cref="IHostBuilder"/> with pre-configured defaults.
+    /// </summary>
     public static class Host
     {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="HostBuilder"/> class with pre-configured defaults.
+        /// </summary>
+        /// <remarks>
+        ///   The following defaults are applied to the returned <see cref="HostBuilder"/>:
```

Comment:
This totally doesn't format right in the docs :(. See the existing [`WebHost.CreateDefaultBuilder`](https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.webhost.createdefaultbuilder?view=aspnetcore-2.2) docs...

If we want a list, we should use [`<list>`](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/xmldoc/list)